src/SML.py

```diff
@@ -1124,9 +1123,6 @@ def __init__(self, type: str, x: np.ndarray = None,
         self.p = model_data['p']
         model_data.close()
 
-        print('\nUsing', self.model_db, 'to predict values...')
-      self.predict()  # Run the model prediction functions
```

Comment:
Why predict during `__init__`
this method should be invoked by the user not during intialization. And even then it will return an error because `self.x.points` is empty

Reply:
Yes, that's true. I was using that line for debugging and testing purposes. It should be removed.